osquery/tables/system/windows/logical_drives.cpp

```diff
@@ -16,53 +16,41 @@ QueryData genLogicalDrives(QueryContext& context) {
   QueryData results;
 
   const WmiRequest wmiLogicalDiskReq(
-      "select DeviceID, DriveType, FreeSpace, Size, FileSystem from "
+      "select DeviceID, Description, FreeSpace, Size, FileSystem from "
       "Win32_LogicalDisk");
-  const std::vector<WmiResultItem>& wmiResults = wmiLogicalDiskReq.results();
-  for (unsigned int i = 0; i < wmiResults.size(); ++i) {
+  auto const& logicalDisks = wmiLogicalDiskReq.results();
+  for (const auto& logicalDisk : logicalDisks) {
     Row r;
-    unsigned int driveType = 0;
     std::string deviceId;
-    wmiResults[i].GetString("DeviceID", deviceId);
-    wmiResults[i].GetUnsignedInt32("DriveType", driveType);
-    wmiResults[i].GetString("FreeSpace", r["free_space"]);
-    wmiResults[i].GetString("Size", r["size"]);
-    wmiResults[i].GetString("FileSystem", r["file_system"]);
+    r["free_space"] = r["size"] = "-1";
```

Comment:
We don't need this here since you're checking this bellow right?

Reply:
Yeah, bad revert. I'll fix this in a moment.

Reply:
Done!